systemvm/debian/opt/cloud/bin/cs/CsProcess.py

```diff
@@ -40,9 +40,9 @@ def kill_all(self):
 
     def find_pid(self):
         self.pid = []
+        items = len(self.search)
         for i in CsHelper.execute("ps aux"):
-            items = len(self.search)
-            proc = re.split(r"\s+", i)[items*-1:]
+            proc = re.split(r"\s+", i)[10:]
```

Comment:
given this is null based the return is now the full cli as returned by `ps aux` ; :+1: